### PR TITLE
Added macOS Terminal font selection instructions; grammar and syntax fixing.

### PR DESCRIPTION
**Pull request by Nobody912**
- Added the macOS Terminal font selection instructions
- Fixed some grammatical/phrasing issues
- Reformatted and adjusted to pass Markdown rules
    - There are still violations of [MD026](https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md026---trailing-punctuation-in-heading) for ending some headers with question marks, no big deal though

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -186,14 +183,7 @@ running Ubuntu 18.04 with the config from the demo.
...
-is fast enough (the threshold where latency becomes noticeable is around 50 ms). You have to be
-careful with Powerlevel9k configuration as it's all too easy to make prompt frustratingly slow.
-Powerlevel10k, on the other hand, doesn't require trading latency for utility -- it's virtually
-instant with any configuration. It stays way below the 50 ms mark, leaving most of the latency
-budget for other plugins you might install.
+In fairness, Powerlevel9k has acceptable latency when given a spartan configuration. If all you need is the current directory without truncation or shortening, Powerlevel9k can render it for you in 17 ms. Powerlevel10k can do the same 30 times faster, but it won't matter in practice because 17 ms is fast enough (the threshold where latency becomes noticeable is around 50 ms) You have to be careful with Powerlevel9k configuration as it's all too easy to make prompt frustratingly slow. Powerlevel10k, on the other hand, doesn't require trading latency for utility — it's virtually instant with any configuration. It stays well below the 50 ms mark, leaving most of the latency budget for other plugins you might install.
```
Comment (romkatv) [inline README.md]: I see that you've removed line splits, turning all paragraphs into long lines. What is the motivation for doing this? Is it against some kind of standard to split long lines into shorter ones?

**Comment by Nobody912** [comment]
Thanks for responding so fast. I love your work!

**Comment by romkatv** [comment]
> I love your work!

Thanks!

So, what's up with line breaks? I'd like to have lines at most 100 chars long. Is this bad in markdown? Do I have to go with unlimited lines and just suck it up?

**Comment by Nobody912** [comment]
Oh my mistake! That is a mistake on my part. Markdown had no rule on line length, but 80-char max length is pretty standard.

**Pull request merged**